hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -191,6 +191,14 @@
       .sinceVersion("0.10.0")
       .withDocumentation("Enable data skipping by collecting statistics once layout optimization is complete.");
 
+  public static final ConfigProperty<Boolean> ROLLBACK_PENDING_CLUSTERING = ConfigProperty
```

Comment:
if this is config based, can we keep this simple and just rollback pending clustering in WriteClient startCommit method. (Don't bother about figuring conflicts between updates )

Reply:
if you strong opinion on keeping conflicts logic, please change config name and description go say clustering rollback will be done only if there is a conflict

Reply:
I think that logic helps when there is pending clustering and no conflicting file group. As a user, I would not want the clustering to be rolled back if there are no conflicts with incoming upsert. I would rather update the docs to be more clear as you suggested.